src/main/conf/quattor-ccm

```diff
@@ -0,0 +1,240 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+# ${build-info}
+
+#
+# Set of tabcompletion function that can be used by other tools to access CCM data
+#
+
+# absolute path to tools, e.g. no fancy ls aliases and unnecessary PATH lookups
+_quattor_find_bin()
+{
+    local dir bin fn
+    fn="$1"
+    # use the "slow" sed here
```

Comment:
OK but - and excuse my ignorance of bash here it's not a language I use - why use sed when bash can do that natively? Or is my version of bash more recent than the average user we want to support?